src/project-tests/pomodoro-clock-tests.js

```diff
@@ -78,476 +84,718 @@ export default function createPomodoroClockTests() {
+      reqNum++;
+      it(`${reqNum}. When a session countdown reaches zero (NOTE: timer MUST 
+      reach 00:00), and a new countdown begins, the element with the id of 
+      "timer-label" should display a string indicating a break has begun.`,
+      function() {
         this.timeout(5000);
         // we decrement session time to the minimum (1 minute)
-        clickButtonsById(Array(60).fill(_sesh_min));
+        clickButtonsById(Array(60).fill(_seshMin));
         // start the pomodoro
-        clickButtonsById([_start_stop]);
+        clickButtonsById([_startStop]);
         return new Promise((resolve, reject) => {
-          const timeLeft = document.getElementById("time-left");
-          const breakLength = document.getElementById("break-length");
+          const timeLeft = document.getElementById('time-left');
+          const breakLength = document.getElementById('break-length');
           // Save label to test that it has changed below
-          let sessionLabel = document.getElementById("timer-label").innerHTML;
+          let tLabelA = document.getElementById('timer-label').innerHTML;
           let shouldBeInBreak = false;
-          const observer = observeElement("time-left", modType => {
-            if (timeLeft.innerHTML === "00:00") {
+          const observer = observeElement('time-left', () => {
+            if (timeLeft.innerHTML === '00:00') {
               shouldBeInBreak = true;
-            } else {
-              if (parseInt(timeLeft.innerHTML.slice(0, 2)) > 5) {
-                reject(new Error("Test timed out because Break time didn't start with the correct value: " + (parseInt(getMinutes(timeLeft.innerHTML)) + 1) + " instead of " + breakLength.innerHTML));
-              } else if (shouldBeInBreak) {
-                if (document.getElementById("timer-label").innerHTML !== sessionLabel) {
-                  resolve();
-                } else {
-                  reject(new Error("Timer has reached zero but didn't switch to Break time"));
-                }
-                observer.disconnect();
+            } else if (parseInt(timeLeft.innerHTML.slice(0, 2), 10) > 5) {
+              reject(
+                new Error('Test timed out because Break time didn\'t start ' +
+                'with the correct value: ' + (
+                  parseInt(getMinutes(timeLeft.innerHTML), 10) + 1
+                ) + ' instead of ' + breakLength.innerHTML)
+              );
+              observer.disconnect();
+            } else if (shouldBeInBreak) {
+              let tLabelB = document.getElementById('timer-label').innerHTML;
+              if (tLabelB !== tLabelA) {
+                resolve();
+              } else {
+                reject(
+                  new Error('Timer has reached zero but didn\'t switch to ' +
+                  'Break time')
+                );
               }
+              observer.disconnect();
```

Comment:
I added observer.disconnect() in both `else if` blocks. Eslint gave the warning: 'no lonely if' on the original code, and it's unclear why. The warning went away when using two else-ifs. I'm not sure when to use observer.disconnect(). Sorry.

Reply:
This is more of a functional concern than a formatting concern. Since you got rid of the ESLint warning and the code still works, let's leave this. I can look at the use of `observer.disconnect()` when we get to the phase of improving the logic for this file. Could you please put a 'TODO' comment directly in the code so we don't forget to look at it?